src/components/Form/Signup/SignupOptions.js

```diff
@@ -99,7 +99,23 @@ const SignupOptions = ({
                 <p>
                     <FormattedMessage id="signup_options_blocktrades" />
                 </p>
-
+                 <a
+                    className="external-link"
+                    href="https://steemwallet.app"
```

Comment:
Does it make sense for this to be an env variable?

Reply:
The rest of the URL's are not, so, I think this is fine for now. Thanks!